app/assets/stylesheets/colors.css.scss

```diff
@@ -1,17 +1,27 @@
-$primary-background-color: rgba(237, 242, 242, 0.8);
-$primary-font-color: rgba(237, 242, 242, 0.8);
-$secondary-font-color: rgba(237, 242, 242, 0.8);
+$primary-background-color: rgba(143, 224, 239, 0.8);
+$secondary-background-color: rgba(176, 196, 200, 0.8);
+$tertiary-background-color: white; 
+$primary-font-color: black;
+$secondary-font-color: gray; 
 
 
+
+.background{
```

Comment:
Don't mix concerns! only put your theme colors in the colors file...
You can use a layouts or main.css file for all the generic css such as title, hr, etc.